pages/products-listing/products.css

```diff
@@ -0,0 +1,87 @@
+.product-list-wrapper {
+  position: relative;
+  color: var(--primary-font-color);
+}
+.product-filters {
+  max-width: 20%;
+  width: 100%;
+  background: var(--primary-background-color);
+  padding: 1.5rem 2.5rem;
+  overflow-x: auto;
+  position: sticky;
+  top: 0rem;
+  height: 100vh;
+}
+.product-filters-heading {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 0;
+}
+.clear-all-cta {
+  text-decoration: underline;
+  text-decoration-color: var(--primary-color);
+  text-underline-position: under;
+  cursor: pointer;
+}
+.clear-all-cta:hover {
+  color: var(--primary-color);
+  font-weight: 500;
+}
+.product-filter-type {
+  padding: 0.75rem 0 0.75rem 0;
+}
+.filter-divider {
+  background-color: var(--filter-divider-color);
+  width: 100%;
+  border: none;
+  height: 0.5px;
+  margin: 0.5rem 0 0.5rem 0;
+}
+.filter-type-heading {
+  font-size: 1.25rem;
+  text-transform: uppercase;
+  padding-bottom: 0.625rem;
+}
+.price-range {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.2rem;
+}
+.slider {
+  width: 100%;
+  cursor: pointer;
+}
+.form-label {
+  display: inline-block;
+  padding-bottom: 0.625rem;
+  box-sizing: border-box;
+  cursor: pointer;
+  font-size: 1.25rem;
+}
+.form-field {
+  margin-right: 0.5rem;
+  width: 1rem;
+  height: 1.5rem;
+  outline: none;
+  display: inline-block;
+  position: relative;
+  top: 0.2rem;
+}
+.product-list {
+  max-width: 80%;
+  width: 90%;
+  padding: 3rem;
+  font-size: 1.8rem;
+  line-height: 1.5;
+}
+.product-list-row {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: space-between;
+  box-sizing: border-box;
```

Comment:
box-sizing: border-box;
try to use this property in reset.css

Reply:
*{box-sizing: border-box;} may be good option

Reply:
Done